### DISCUSSION TITLE
如何添加点播服务网站？

### DISCUSSION THREAD

**Discussion by e-linc** [gaotianliuyun/gao#218] [Q&A]
你好！新手有问题想请教。

我在一个安卓4.4的小米盒子（型号：MDZ-09-AA）上，使用俊版的TVBox，搭配他的配置接口，直播很好，但点播搜索出来的结果只有泥巴、360和阿里云盘。泥巴只能在墙外看，阿里云盘的链接都是”找不到数据“，而360就一直”正在嗅探播放地址“然后”嗅探失败“……我用OK接口点播时搜索结果还有小苹果等其它视频网站可播放，就尝试把OK接口JSON文件中的“sites”一节下的小苹果条目复制到俊佬的配置文件里，但搜索结果里依然没有小苹果。我想请教是哪里出了问题，能否请你介绍自行融合、修改JSON配置文件要注意的地方？

谢谢！

**Comment by e-linc** [comment]
讨论区没人吗？

**Comment by gaotianliuyun** [comment]
一个很好的播放源，需要jar的支持，需要json文件的正确配置，最重要一点还需要壳的支持。
据我所知，OK的jar只支持影视壳，不支持tvbox的壳。另外小苹果好像是爬虫，你在其他配置里加入的话，需要外挂jar。

**Reply by e-linc** [reply]
谢谢你的回复和解释！

我在俊版的TVBox应用里用OK的接口好像可以啊，不知道是不是误解了你的意思。我更倾向于俊版接口的原因是直播——OK的直播线路虽然多但在俊版的TVBox应用上只有一条可用，有时稳定性也不如俊版接口。

我在你仓库的jar目录里没找到小苹果的外挂jar，不知道是不是没找对地方？或者能否请你推荐TVBox壳上好用的配置？我试过你上一版的0821.json（记得当时有标注“大而全”），提示“配置解析失败”，不知道是不是对安卓版本有限制？

新手问题多，麻烦你了！

**Reply by gaotianliuyun** [reply]
我看饭太硬的jar里写有小苹果的爬虫，如果你想在PG里的加上小苹果的配置。可以加上：
{"key":"苹果","name":"苹果┃不卡","type":3,"api":"csp_LiteApple","searchable":1,"quickSearch":1,"changeable":1,"jar":"./jar/fan.txt;md5;2a085160a4140e1697c5d6393901ec7e"},

我是不用TVBox的，所以我不能更多给你推荐，你可以试试我自述文件推荐的配置。比如饭太硬、王二小等。
